testutil/singlevm/setup.sh

```diff
@@ -154,14 +155,17 @@ function createWorkloads() {
 	EOF
 	) > "$ciao_bin"/workload_examples/fedora_vm.yaml
 
+	# get the clear image id
```

Comment:
@kaccardi with https://github.com/01org/ciao/pull/1004 you will have suffix matching, which will make this simpler